src/_test_/Continent.test.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { Provider } from 'react-redux';
+import renderer from 'react-test-renderer';
+import { BrowserRouter } from 'react-router-dom';
+import store from '../Redux/configureStore';
+import '@testing-library/jest-dom';
+import Continent from '../components/Continent';
+
+describe('Continent component', () => {
+  test('renders correctly', () => {
+    const tree = renderer.create(
+      <Provider store={store}>
+        <BrowserRouter>
```

Comment:
- [ ] Good job adding tests on the correct render of your pages but please add also unit tests for several components of the app and some pure functions in the Redux code.